micrometer/src/main/java/feign/micrometer/MeteredClient.java

```diff
@@ -40,7 +43,21 @@ public MeteredClient(Client client,
       MeterRegistry meterRegistry,
       MetricName metricName,
       MetricTagResolver metricTagResolver) {
-    this.client = client;
+    this.delegate = client;
+    this.meterRegistry = meterRegistry;
+    this.metricName = metricName;
+    this.metricTagResolver = metricTagResolver;
+  }
+
+  public MeteredClient(AsyncClient<Object> asyncClient, MeterRegistry meterRegistry) {
```

Comment:
Could you also do the same for dropwizard implementations?!

Reply:
Yep, I was testing the waters with micrometer one.